### ISSUE TITLE
[Bug] iOS CollectionView's Scrolled event can not be trigger when scrolling in HeaderTemplate 

### ISSUE THREAD

**Issue by huyjack178** [xamarin/Xamarin.Forms#8548]
### Description
I use scrolled event of collection view for detecting scroll offset, then hide the StackLayout when scrolling up for more space at bottom. But when I create a HeaderTemplate which has HeightRequest bigger than current screen's height, the Scrolled event can not be fired, so I can not scroll down for showing the hidden layout. The ItemsViewScrolledEventArgs.VerticalOffset can not be back to zero. Take a look at before and after screenshot

### Basic Information

- Version with issue: Xamarin.Forms 4.3.0.947036
- iOS:  13.2
- Nuget Packages:

### Screenshots
Before scroll down
![image](https://user-images.githubusercontent.com/11793001/69049380-4d50b980-0a32-11ea-8635-0423fcfd5289.png)

After, can not scroll down to view the top layout

![image](https://user-images.githubusercontent.com/11793001/69049399-5772b800-0a32-11ea-8e4c-30d490dd1d3a.png)

**Comment by StephaneDelcroix** [comment]
could you please provide a simple sample showcasing your issue, so we can reproduce, confirm, and work on a fix ? thanks.

**Comment by huyjack178** [comment]
@StephaneDelcroix Here you are, also update the issue's description. There is 2 tab, first tab collection view has no header, the second tab has full height header and issue happened
https://drive.google.com/file/d/1Ug5Hzr2tw947UyDU6SZjqUOdTcMqdFrs/view?usp=sharing

**Comment by huyjack178** [comment]
@StephaneDelcroix Also has other issue, the refreshView loading indicator draw behind the Header

**Comment by jfversluis** [comment]
Hi @huyjack178 thanks for reporting this. I'm not entirely sure if I understand what the issue is.

First of; for the `RefreshView` bug, please see if there is already an issue for it and if not open a separate one so we can stay focused on one issue at a time :)

For this issue, I have played around with your reproduction a little but the scrolled event seems to trigger just fine. Although I do see the `StackLayout` is not coming back into view entirely when I scroll down again.

What I also noticed is that you are using a `CollectionView` within a `CollectionView` which might be the cause of some weird behavior here. Is that also how you are using it in your actual app?

Long story short: would you be able to try and explain a bit more about what is going wrong according to you?

**Comment by huyjack178** [comment]
Hi @jfversluis , you would see I have 2 pages, one page have the CollectionView's Header with bigger height than screen height, I when I scroll it up, the Scrolled event can not be triggered. You can create a breakpoint in Scrolled event method inside xaml.cs file
